contrib/dracut/90zfs/mount-zfs.sh.in

```diff
@@ -60,14 +60,12 @@ if import_pool "${ZFS_POOL}" ; then
 	if [ $(zpool list -H -o feature@encryption $(echo "${ZFS_POOL}" | awk -F\/ '{print $1}')) == 'active' ]; then
 		# if the root dataset has encryption enabled
 		ENCRYPTIONROOT=$(zfs get -H -o value encryptionroot ${ZFS_DATASET})
-		if [[ $(echo -n "${ENCRYPTIONROOT}" | grep -v '^-$' | grep -v off ]]; then
+    if [ $(echo "${ENCRYPTIONROOT}" | grep -v -E '^$|^-$') ]; then
```

Comment:
I still don't know why this check is this way, @tcaputi can encryptionroot ever be empty? mine only shows up as `-`

the check I'm using on my system is just:

```
if ! [ "${ENCRYPTIONROOT}" == "-" ]; then
```

Reply:
I like @kpande's check better. Especially since an encryptionroot with "off" in the name would mess with the script. It should only ever be a dataset name or "-"

Reply:
the reason my check was needed is because echo outputs a newline

Reply:
no need to use echo at all

Reply:
and it currently doesnt

Reply:
but the PR won't update because it was closed

Reply:
wonder why that happened

Reply:
is it standard procedure to close a PR and resubmit with minor modifications without working with the person first?

Reply:
if so, then I'm done working upstream here

Reply:
No it's not.  @prometheanfire I'm sorry about that.  We only hand off issues after taking with the original author.